### PR TITLE
Mention FAQ guide for help with issues like

### PR DESCRIPTION
**Pull request by wjimenez5271**

### CODE REVIEW

**Comment by untergeek** [comment]
Please sign the [CLA](http://elastic.co/contributor-agreement/).  Yes, it may seem silly, but even documentation changes have copyrights.

File: README.md [hunk 1]
```diff
@@ -26,6 +26,10 @@ Install using pip
 
 Run `curator --help` for usage specifics.
 
+### Common Issues
```
Comment (untergeek) [inline README.md]: Rather than make users feel like there's "common issues," perhaps this would be better titled, **Frequently Asked Questions**

Comment (untergeek) [inline README.md]: And you could link that entire header to http://www.elastic.co/guide/en/elasticsearch/client/curator/current/faq.html

File: README.md [hunk 2]
```diff
@@ -26,6 +26,10 @@ Install using pip
...
+
+Encountering issues like `DistributionNotFound`? See the [FAQ](http://www.elastic.co/guide/en/elasticsearch/client/curator/current/entrypoint-fix.html) for help.
```
Comment (untergeek) [inline README.md]: And I like this here!  Go to the FAQ!

**Comment by wjimenez5271** [comment]
@untergeek no worries, just signed it

**Comment by untergeek** [comment]
Thanks for signing.  Any future PRs will not have the warning.

**Comment by untergeek** [comment]
I love it!  Thanks for contributing.  Before I merge, feel free to add yourself to the CONTRIBUTORS file, following the pattern there.

**Comment by wjimenez5271** [comment]
@untergeek oh cool, thanks!

**Pull request merged**